pubnub/lib/src/crypto/aesCbcCryptor.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:encrypt/encrypt.dart' as crypto;
+import 'package:crypto/crypto.dart' show sha256;
+import 'dart:typed_data' show Uint8List;
+
+import 'package:pubnub/core.dart';
+
+class AesCbcCryptor implements ICryptor {
+  static const BLOCK_SIZE = 16;
```

Comment:
Is it ok to make this var private? Do you access it outside `AesCbcCryptor`?
I also noticed you use it only once, so it should be possible to replace it with the local variable in this way: 

```
 crypto.IV _getIv() {
    final aesBlockSize = 16
    return crypto.IV.fromSecureRandom(aesBlockSize);
  }
```

It's rather nitpick, so you can also leave it as it is :)

Reply:
True,
addressed in recent push.